src/apps/backend/modules/comment/internal/store/comment_model.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import dataclass, field
+from datetime import datetime, timezone
+from typing import Optional
+
+from bson import ObjectId
+
+
+@dataclass
```

Comment:
`@dataclass(frozen=True)` to make sure its members are not mutable once constructed